src/main/scala/robosky/ether/world/feature/minidungeons/MinidungeonFeatureConfig.scala

```diff
@@ -0,0 +1,8 @@
+package robosky.ether.world.feature.minidungeons
+
+import java.util.Optional
+
+import net.minecraft.util.Identifier
+
+// Does not extend FeatureConfig so we can get it into the StructureStart because this shit is dumb
+case class MinidungeonFeatureConfig(name: String, template: Identifier, loot: Optional[Identifier])
```

Comment:
Why Java's Optional

Reply:
Because I need to get a value out from java code elsewhere, I hate it too